app/ui-react/packages/ui/src/Data/Virtualizations/PublishStatusWithProgress.tsx

```diff
@@ -36,25 +39,72 @@ export const PublishStatusWithProgress: React.FunctionComponent<
           }
           i18nLogUrlText={props.i18nPublishLogUrlText}
         />
-        {props.publishVersion
-          ? '(version: ' + props.publishVersion + ') '
-          : null}
-        {props.modified ? ' M ' : null}
+        <span className={'publish-status-with-progress_text'}>
+          {props.publishVersion
+            ? ' (Version' + props.publishVersion + ') '
+            : null}
+        </span>
+        {props.modified ? (
+          <Badge key={1} className={'publish-status-with-progress-badge'}>
+            Modified
+          </Badge>
+        ) : null}
       </div>
     );
   }
+  if (props.inListView) {
+    return (
+      <React.Fragment>
+        <Label
+          className={'publish-status-with-progress_Label'}
+          type={props.labelType}
+        >
+          {props.i18nPublishState}
+        </Label>
+        {props.publishVersion ? (
+          <span className={'publish-status-with-progress_text'}>
+            {' (Version' + props.publishVersion + ') '}
+          </span>
+        ) : (
+          // tslint:disable-next-line: jsx-self-close
+          <span className={'publish-status-with-progress_version-container'} />
+        )}
+        {props.modified ? (
+          <Badge key={2} className={'publish-status-with-progress-badge'}>
+            Modified
+          </Badge>
+        ) : (
+          // tslint:disable-next-line: jsx-self-close
+          <span className={'publish-status-with-progress-badge-container'} />
+        )}
+      </React.Fragment>
+    );
+  }
 
   // no progress bar needed so just show a label
   return (
     <React.Fragment>
-      <Label
-        className={'publish-status-with-progress__label'}
-        type={props.labelType}
-      >
+      {props.i18nPublishState === 'Running' ? (
+        <OkIcon
+          color={'#49B720'}
+          height={'1.25rem'}
+          width={'1.25rem'}
+          className={'publish-status-with-progress-margin10'}
+        />
+      ) : (
+        ''
+      )}
+      <span className={'publish-status-with-progress_text'}>
         {props.i18nPublishState}
-      </Label>
-      {props.publishVersion ? '(version: ' + props.publishVersion + ') ' : null}
-      {props.modified ? ' M ' : null}
+        {props.publishVersion
```

Comment:
U could do 

```
{props.publishVersion && (`(Version ${props.publishVersion})`)}
```

the template string beats concatenation any day, and avoiding the extra whitespace that `?` imposes on you is nice.

Reply:
sure will improve this in next pr and will keep in mind for future changes